src/components/02-elements/anchors/anchors.scss

```diff
@@ -7,3 +7,9 @@ a {
   &:hover { color: hsla(var(--orange-dark-hsl), 1);; }
   &:active { color: hsla(var(--yellow-dark-hsl), 1);; }
 }
+
+a[target="_blank"]::after {
+  content: " (Opens in new tab)";
+  visibility: hidden;
+  voice-over: hidden;
```

Comment:
I've not seen this `voice-over` CSS property before. Can you point to some documentation?

Reply:
Thanks yeah, copy paste error.